oc_cdtapi/DmsGetverAPI.py

```diff
@@ -184,6 +317,88 @@ def get_dms_log_url(self, distr_id, distr_option):
 
         return url
 
+    def json_or_none(self, resp):
+        """
+        Returns requests response's json if status code is 200-299 or logs error and returns None
+        :param requests.response resp: requests' response
+        :return: requests.response.json or None
+        """
+        logging.debug('Reached json_or_none')
+        status_code = resp.status_code
+        if 200 <= status_code <= 299:
+            logging.debug('[%s] status code received, trying to return json' % status_code)
+            return resp.json()
+        else:
+            logging.error('[%s] status code received, assuming an error, returning None' % status_code)
+        return None
+
+    def login(self):
+        """
+        Obtain auth token
+        """
+        logging.debug('Reached login')
+        username = os.getenv('DBSM2_USER')
+        password = os.getenv('DBSM2_PASSWORD')
```

Comment:
This is the place where I am hardly sure we have to split *DmsGetver* and *DBSM* classes.
Too many differences, including authorization type.

Reply:
split, now it's Dbsm2API